try {
   ...
} catch (ExceptionClassName  referenceName) {
   ...
} finally {	
   ...
}




try {
   ...
} finally {	
   ...
}
